### DISCUSSION TITLE
Feature Request: run Tiptap editor in Nodejs

### DISCUSSION THREAD

**Discussion by martinboksa** [ueberdosis/tiptap#4050] [Feature Requests]
### Description

It would be helpful to be able to execute Tiptap in Node.js.

### Current workaround
For example; this works when I need `HTML`

```
      const { extensions } = [....] // tiptap extensions
      const schema = getSchemaByResolvedExtensions(extensions) //
      const contentNode = Node.fromJSON(schema, feedback)

      const { JSDOM } = jsdom
      const dom = new JSDOM('<!DOCTYPE html><div id="content"></div>')
      const doc = dom.window.document
      const target = doc.querySelector('div')!

      DOMSerializer.fromSchema(schema).serializeFragment(
        contentNode.content,
        {
          document: doc,
        },
        target
      )

      console.log(doc.getElementById('content')!.innerHTML)
```

### Use Case

I need to perform some postprocessing on data where I am supposed to call `editor.getText()` or `editor.getHTML()`

### Type

New feature

**Comment by blackcoffee2** [comment]
You can use [engine](https://github.com/blackcoffee2/tiptap-engine) I wrote for tiptap.
My use case was to be able to run unit tests in node.js which requires running tiptap in non-browser environment.

It has support for `editor.getText()` and `editor.getHTML()`.

Link to the API docs is [here](https://github.com/blackcoffee2/tiptap-engine/blob/main/API.md#getcontent)
